src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -381,9 +414,40 @@ export default {
 		if (this.emojiTribute) {
 			this.emojiTribute.detach(this.$el)
 		}
+		if (this.linkTribute) {
+			this.linkTribute.detach(this.$el)
+		}
 	},
 
 	methods: {
+		getLink(item) {
```

Comment:
I find it a bit confusing to have `this.getLink()` and global `getlink()` in the component, but that's only nitpicking.